coralproject/talk
# [CORL-1535] Production Build with Profiler

## What does this PR do?
-  Implements `npm run build:withProfiler`

## How do I test this PR?
- Build client and server using  `npm run build:withProfiler`.
- Go to `http://localhost:3000/embed/stream?storyURL=<Your Story URL>` and see that the React Dev Tools including the profiler works!
